src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4653,6 +4653,8 @@ internal static SessionStateAliasEntry[] BuiltInAliases
                         "New-Module",    "", ScopedItemOptions.ReadOnly | ScopedItemOptions.AllScope),
                     new SessionStateAliasEntry("oh",
                         "Out-Host",        "", ScopedItemOptions.ReadOnly | ScopedItemOptions.AllScope),
+                    new SessionStateAliasEntry("ral",
+                        "Remove-Alias", "", ScopedItemOptions.ReadOnly | ScopedItemOptions.AllScope),
```

Comment:
Sorry for being late - for new cmdlet we should add the alias by [Alias()] attribute before `public class RemoveAliasCommand : PSCmdlet`.

Reply:
I added the Alias using [Alias()]. Does this make the
```PowerShell
new SessionStateAliasEntry("ral", 
"Remove-Alias", "", ScopedItemOptions.ReadOnly | ScopedItemOptions.AllScope), 
```
obsolete?

Reply:
Yes, please remove.